coordinator/src/main/scala/com/socrata/datacoordinator/service/CoordinatorErrorsAndMetrics.scala

```diff
@@ -34,6 +36,14 @@ class CoordinatorErrorsAndMetrics(formatDatasetId: DatasetId => String) extends
       "commandIndex" -> JNumber(commandIndex))
   }
 
+  def fieldNameErrorResponse(msg: String, commandIndex: Long, dataset: DatasetId, fieldName: ColumnName, resp: HttpResponse = BadRequest) = {
+    import scala.language.reflectiveCalls
```

Comment:
I remember needing it? But commenting it out... it appears to work. So I will remove that.